### PR TITLE
[ci skip] Fix spelling of ignition's word into javadoc (#11862)

### PR DESCRIPTION
**Pull request by x9nico**
Just a small fix into javadoc of Attribute's class for spelling the ignition's word caused by Spigot.

### CODE REVIEW

**Comment by Doc94** [comment]
The Paper comments are not needed with hardfork 😌

**Comment by x9nico** [comment]
> The Paper comments are not needed with hardfork 😌

Oh ok, I fixed it.

File: paper-api/src/main/java/org/bukkit/attribute/Attribute.java
Comment (masmc05) [inline paper-api/src/main/java/org/bukkit/attribute/Attribute.java]: Those spaces are kinda out of place, either the whole file needs to receive them (most probably bad without a maintainer's approval), or remove here the diff that adds them

Comment (x9nico) [reply paper-api/src/main/java/org/bukkit/attribute/Attribute.java]: These spaces was due to my old Paper comments (which was uselesss), I fixed the spaces. Thanks.

Comment (x9nico) [reply paper-api/src/main/java/org/bukkit/attribute/Attribute.java]: > Those spaces are kinda out of place, either the whole file needs to receive them (most probably bad without a maintainer's approval), or remove here the diff that adds them

These spaces was due to my old Paper comments (which was uselesss), I fixed the spaces. Thanks.

**Review by Machine-Maker — APPROVED** [review]

**Review by lynxplay — APPROVED** [review]

**Pull request merged**
